src/client/scripts/esm/chess/variants/variant.ts

```diff
@@ -359,7 +358,7 @@ function getGameRulesOfVariant({ Variant, UTCDate = timeutil.getCurrentUTCDate()
 	if (variantEntry.gameruleModifications?.hasOwnProperty(0)) { // Multiple UTC timestamps
 		gameruleModifications = getApplicableTimestampEntry(variantEntry.gameruleModifications, { UTCDate, UTCTime });
 	} else { // Just one gameruleModifications entry
-		gameruleModifications = variantEntry.gameruleModifications;
+		gameruleModifications = variantEntry.gameruleModifications as GameRules;
```

Comment:
Is this a typo? Is it supposed to be GameRuleModifications?

Reply:
I believe so, yes.